packages/babel-plugin-transform-runtime/package.json

```diff
@@ -11,6 +11,10 @@
   "keywords": [
     "babel-plugin"
   ],
+  "browser": {
```

Comment:
Seems like it suffices to set [`preferBuiltins: false`](https://github.com/rollup/plugins/tree/master/packages/node-resolve#preferbuiltins) when building babel-standalone, so it can warn us on builtin imports.

Reply:
Oh but it will not cover cases like `resolve` is imported. Maybe we can do something like
```js
{
  name: "warn-browser-unsupported-modules",
  resolveId(...args) {
    const [source] = args;
    if (["resolve"].includes(source)) {
      throw new Error(`${source} can not be bundled into @babel/standalone!`);
    }
    return this.resolve(...args);
  }
}
```

Reply:
Yeah, I'm merging this for now and I will investigate. I'm surprised that I can't find a plugin that does what we need :thinking:

Reply:
@nicolo-ribaudo You may checkout https://github.com/dumbmatter/rollup-plugin-blacklist but I am not strongly for that.

Reply:
May I ask why don't you like it?